CarRentalCompany.Integration.Volvo/Factories/VolvoReceiptFormFactory.cs

```diff
@@ -0,0 +1,23 @@
+﻿using CarRentalCompany.Domain.Models;
+using CarRentalCompany.Integration.Builders;
+using CarRentalCompany.Integration.Factories;
+
+namespace CarRentalCompany.Integration.Volvo.Factories
+{
+    public class VolvoReceiptFormFactory : ICarReceiptFormFactory
+    {
+        public static string Type 
+            => "Volvo";
+
+        public CarReceiptForm Apply(CarReceiptFormBuilder builder, IEnumerable<ActivityDefinition> activities)
+        {
+            var activitiesInstances = activities
```

Comment:
czemu te wszystkie Apply wygladaja doklanie tak samo? Cos tu nie gra

Reply:
Nie rozumiem po co te integracje i formFactory skoro one wszystkie robią dokładnie to samo